astrocut/__init__.py

```diff
@@ -28,3 +28,4 @@ class UnsupportedPythonError(Exception):
     from .cutout_processing import (path_to_footprints, center_on_path,  # noqa
                                     CutoutsCombiner, build_default_combine_function)  # noqa
     from .asdf_cutouts import asdf_cut, get_center_pixel  # noqa
+    from .cube_cut_from_footprint import cube_cut_from_footprint  # noqa
```

Comment:
this hides the module, is that intended?

I think it would be better to name the module differently from the function that you are intending to export

Reply:
Good point! I renamed the module to `footprint_cutouts`. 

I'm not sure if I know what you mean by hiding the module. You can still import the module on its own.

Reply:
without the change to the name of the module, you couldn't easily import the module from another file. 

you had to resort to some trickery. e.g.:

```py
import astrocut.cube_cut_from_footprint

ccff = sys.modules["astrocut.cube_cut_from_footprint"]
```

Reply:
note that the first import above doesn't import the module, it imported the function. but the module does get added to sys.modules when you import the function, so you can extract it from there

Reply:
I see, that makes sense!